_pages/stp.md

```diff
@@ -1,49 +1,108 @@
 ---
+2
 layout: people
+3
 title: Scientist Training Programme (STP) Scheme
+4
 permalink: /stp.html
+5
 description:
+6
 ---
-
+7
+​
+8
 The NHS employs Clinical Scientists who work in scientific roles within hospitals. The NHS Scientist Training Programme 
+9
 (STP) is a three-year [scheme](https://nshcs.hee.nhs.uk/programmes/stp/) of work-based learning, supported by a 
+10
```

Comment:
Weirdly, there were [zero width-spaces](https://en.wikipedia.org/wiki/Zero-width_space) here and in other lines - maybe related to the formatting/line numbers? I don't think it actually affects how the page looks, though.

<img width="248" alt="image" src="https://user-images.githubusercontent.com/67117138/178591740-6da60aa8-cc78-4362-b04b-30077929f7fe.png">